monitoring/osd-managing-alerts.adoc

```diff
@@ -0,0 +1,8 @@
+[id="alerts"]
+= Alerts
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: alerts
+
+toc::[]
+
+Alerts for user-defined projects are not currently supported on this product.
```

Comment:
missed Next steps section, it links to Troubleshooting monitoring issues

Next steps
. Troubleshooting monitoring issues

Reply:
Done.